lib/executor.js

```diff
@@ -10,7 +10,7 @@ var Executor = function (capturedBrowsers, config, emitter) {
     var nonReady = []
 
     if (!capturedBrowsers.length) {
-      log.warn('No captured browser, open http://%s:%s%s', config.hostname, config.port,
+      log.warn('No captured browser, open ' + (config.protocol || 'http:') + '//%s:%s%s', config.hostname, config.port,
```

Comment:
Please move the default setting into `lib/config.js`, so we don't have to add these checks everywhere